runner.rb

```diff
@@ -1,9 +1,9 @@
 require './lib/stat_tracker'
 
 
-game_path = './data/games.csv'
+game_path = './data/games_dummy.csv'
```

Comment:
Lets do this on the spec fil vs runner so that our tests run faster but our runner is still the full file for the project

Reply:
Thank you!